Tools/winiot/sync_win_iot.py

```diff
@@ -0,0 +1,310 @@
+import argparse
```

Comment:
This is copied from a file that no longer exists in the main repo, so ideally we'd move whatever specialized functionality is in here into `PC/layout/main.py`

Reply:
added --preset-iot, and copy_if_modified to pc/layout.  
Can use --zip <file> parameter to produce files to use in docker images.